ffi/mxcfb_kindle_h.lua

```diff
@@ -56,62 +56,62 @@ enum mxcfb_dithering_mode {
   EPDC_FLAG_USE_DITHERING_MAX = 5,
 };
 struct mxcfb_rect {
-  unsigned int top;
-  unsigned int left;
-  unsigned int width;
-  unsigned int height;
+  uint32_t top;
+  uint32_t left;
+  uint32_t width;
+  uint32_t height;
```

Comment:
Is this a "real" change or some side effect from ffi-cdecl btw?

Reply:
That was me switching to C99 types in the headers instead of Linux typedefs, which ffi-cdecl just resolved to base types instead of using the matching C99 ones ;).